README.md

```diff
@@ -130,3 +130,25 @@ Check `sideEffect/saga.js` for a detail of the inputs and outputs
 * Add unit tests
 * Add `<ReferenceMany>` field
 * Add complex field & input types
+
+## Contributing
```

Comment:
Let's keep that in the README for now, `Contributing.md` is more a guideline for pull requests. At that step, it's probable that contributors will have already tested their changes (I hope).